packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -440,6 +440,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
     }
 
     protected serverDisconnectHandler = (error: IFluidErrorBase & OdspError) => {
+        this.logger.sendTelemetryEvent({ eventName: "ServerDisconnect" }, error);
```

Comment:
@jatgarg - having clientId here would make it so much easier to analyze, as we would be able to easily pair Container's disconnect events with these events across all sessions and find all the sessions where we do have this event, but disconnect reason is "transport error".
It's possible without this data, but much harder / more manual.